docs/changelogs/CHANGELOG-2.22.md

```diff
@@ -0,0 +1,481 @@
+# Kubermatic 2.22
+
+- [v2.22.0](#v2220)
+
+## [v2.22.0](https://github.com/kubermatic/kubermatic/releases/tag/v2.22.0)
+
+Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.22/tutorials-howtos/upgrading/). Consider tweaking `seedControllerManager.maximumParallelReconciles` to ensure user cluster reconciliations will not cause resource exhaustion on seed clusters.
+
+### Supported Kubernetes Versions
+
+- Add support for Kubernetes 1.24.8. Those Kubernetes patch releases fix CVE-2022-3162 and CVE-2022-3294, both in kube-apiserver. We strongly recommend upgrading to those Kubernetes patch releases as soon as possible ([#11340](https://github.com/kubermatic/kubermatic/pull/11340))
+    - [CVE-2022-3162: Unauthorized read of Custom Resources](https://groups.google.com/g/kubernetes-announce/c/oR2PUBiODNA/m/tShPgvpUDQAJ)
+    - [CVE-2022-3294: Node address isn't always verified when proxying](https://groups.google.com/g/kubernetes-announce/c/eR0ghAXy2H8/m/sCuQQZlVDQAJ)
+- Add support for Kubernetes v1.24.9 ([#11553](https://github.com/kubermatic/kubermatic/pull/11553))
+- Add support for Kubernetes 1.25 ([#11049](https://github.com/kubermatic/kubermatic/pull/11049))
+- Add support for Kubernetes 1.26 ([#11621](https://github.com/kubermatic/kubermatic/pull/11621))
+- Support for Kubernetes 1.22 and 1.23 user clusters has been removed; User clusters remaining on 1.23 will be automatically upgraded with this KKP version ([#11286](https://github.com/kubermatic/kubermatic/pull/11286), [#11767](https://github.com/kubermatic/kubermatic/pull/11767))
+- Allow Kubernetes version upgrade for clusters with non-amd64 nodes & Canal CNI and IPVS for all Kubernetes versions ([#11765](https://github.com/kubermatic/kubermatic/pull/11765))
+- Add support for Kubernetes v1.24.10, v1.25.6, and v1.26.1 ([#11859](https://github.com/kubermatic/kubermatic/pull/11859))
+
+#### Supported Versions
+
+- 1.24.8
+- 1.24.9
+- 1.24.10
+- 1.25.4
+- 1.25.6
+- 1.25.6
+- 1.26.1
+
+### Highlights
```

Comment:
Let's also add the new Admin panel and Project panel sidebars (the groupings and toggle feature makes the feature list more structured)

Reply:
- Including MLA within KKP installer could also be highlighted here as well
- Cilium CNI customization and management on Applications base could also be

Reply:
I've added CNI management via Applications. While MLA integration in the installer is nice, it doesn't change much for day-to-day usage and we're highlighting this change on the upgrade guide already.